docs/providers/aws/events/alexa-smart-home.md

```diff
@@ -0,0 +1,43 @@
+<!--
+title: Serverless Framework - AWS Lambda Events - Alexa Smart Home
+menuText: Alexa Smart Home
+menuOrder: 11
+description:  Setting up AWS Alexa Smart Home Events with AWS Lambda via the Serverless Framework
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/providers/aws/events/alexa-smart-home)
+<!-- DOCS-SITE-LINK:END -->
+
+# Alexa Smart Home
+
+## Event definition
+
+This will enable your Lambda function to be called by an Alexa Smart Home Skill.
+`amzn1.ask.skill.xx-xx-xx-xx` is an application ID for Alexa Smart Home. You need to sign up [Amazon Developer  Console](https://developer.amazon.com/) and get your application ID.
```

Comment:
`Amazon Developer  Console` --> `Amazon Developer Console` (one space can be removed 😬).